cmd/kops/validate_cluster.go

```diff
@@ -171,7 +171,8 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 	} else {
 		// do we need to print which instance group is not ready?
 		// nodes are going to be a pain
-		fmt.Fprint(out, "\nValidation Failed\n")
+		// FIXME: format still not making sense
```

Comment:
Mostly referring to 

> Stop making sense

I don't know why are are putting in `FIXME:`'s (is that even a thing?) in the code? What is is broken here?